package.json

```
@@ -110,7 +110,7 @@
     "extensionizer": "^1.0.1",
     "fast-json-patch": "^2.0.4",
     "fuse.js": "^3.2.0",
-    "gaba": "^1.6.0",
+    "gaba": "1.7.0",
```

```suggestion
    "gaba": "^1.7.0",
```

Generally it's better to use version ranges rather than pinning, as the lockfile ensures it won't update unexpectedly.

---

definitely didn't mean to pin it, good catch